src/element.ts

```diff
@@ -491,4 +483,95 @@ export abstract class Element {
     const fontInfo = this.fontInfo;
     this.setFont(fontInfo.family, fontInfo.size, weight, fontInfo.style);
   }
+
+  /** Get element width. */
+  getWidth(): number {
+    return this.width;
+  }
+
+  /** Set element width. */
+  setWidth(width: number): this {
+    this.width = width;
+    return this;
+  }
+
+  /** Shift element down `y` pixels. Negative values shift up. */
+  setYShift(y: number): this {
+    this.yShift = y;
+    return this;
+  }
+
+  /** Get shift element `y` */
+  getYShift(): number {
+    return this.yShift;
+  }
+
+  /**
+   * Set shift element right `x` pixels. Negative values shift left.
+   */
+  setXShift(x: number): this {
+    this.xShift = x;
+    return this;
+  }
+
+  /** Get shift element `x` */
+  getXShift(): number {
+    return this.xShift;
+  }
+
+  /**
+   * Set element text
+   */
+  setText(txt: string): this {
+    this.text = txt;
+    return this;
+  }
+
+  /** Get element text */
+  getText(): string {
+    return this.text;
+  }
+
+  /**
+   * Render the element text .
+   */
+  renderText(ctx: RenderContext, xPos: number, yPos: number): void {
+    ctx.save();
+    ctx.setFont(this.textFont);
+    ctx.fillText(this.text, xPos + this.xShift, yPos + this.yShift);
+    ctx.restore();
+  }
+
+  /** Canvas used to measure text */
+  protected static txtCanvas?: HTMLCanvasElement;
```

Comment:
Is this an internal member? Should it use the private `#` syntax? See fraction.ts:

```
static #fractionA = new Fraction();
```

Reply:
Done